karapace/protobuf/io.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+# -*- coding: utf-8 -*-
+
+from io import BytesIO
+from karapace import config
+from karapace.protobuf.exception import IllegalArgumentException, ProtobufSchemaResolutionException, ProtobufTypeException
+from karapace.protobuf.message_element import MessageElement
+from karapace.protobuf.protobuf_to_dict import dict_to_protobuf, protobuf_to_dict
+from karapace.protobuf.schema import ProtobufSchema
+from karapace.protobuf.type_element import TypeElement
+from typing import Any, List
+
+import hashlib
+import importlib
+import importlib.util
+import logging
+import os
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Was this intentional? Do you want to have this license block here?

Reply:
no, need to remove

Reply:
I meant it is in the middle of the import block, why not at the top of the file? This position may have issues with isort

Reply:
I mostly not use License manifests in this code. I suppose I forgot remove it from code which I get as template and imports above was added by some pycharm/mypy dependency resolving tool